wordle-web/src/router/index.ts

```diff
@@ -1,19 +1,17 @@
 import { createRouter, createWebHistory } from 'vue-router'
-import HomeView from '../views/HomeView.vue'
-import WordleView from '../views/WordleView.vue'
 
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
       path: '/',
       name: 'home',
-      component: HomeView
```

Comment:
I'm not sure how much it really matters, but I remember Grant saying a few classes ago that the format you had before is the better way to set up the route. Where you import above instead of in the same line.